### DISCUSSION TITLE
Changes to folders are not reflected instantly

### DISCUSSION THREAD

**Discussion by kabeza** [elementary/files#2278] [General]
Hi
I've latest Elementary OS with latest updates, etc. 

I'm noticing lately that any change made to a folder (even folders from home/myuser), whether it's downloading a new file, renaming a file, pasting copied files/folders, etc. they are not reflected instantly, instead I have to press F5 for those changes to be updated. 
What could be the cause of this inconvenience?

Thanks

**Comment by jeremypw** [comment]
This sounds similar to issue #2179, for which a fix has just been merged. So this should be fixed in the next release.

**Reply by kabeza** [reply]
Yes, it is that, same behaviour

I guess I have the latest version (which now as day of writing is 6.4.1) so I'll wait for next release

```
apt list --installed | grep pantheon-files
WARNING: apt does not have a stable CLI interface. Use with caution in scripts.
libpantheon-files-core0/jammy,now 6.4.1+r5948+pkg120~ubuntu7.1 amd64 [instalado, automático]
pantheon-files/jammy,now 6.4.1+r5948+pkg120~ubuntu7.1 amd64 [instalado, automático]

```

Thanks again

**Comment by jeremypw** [comment]
There is another discussion open about refreshing the view.  Please continue the discussion there or open a specific focused issue that can be addressed individually.
